python/setup.py.in

```diff
@@ -145,6 +146,7 @@ class Configure(build_ext):
         # pkg-config for a minimum version 0. If it's installed, it should
         # return True and we'll use it. Otherwise, we'll use the stub.
         if misc.pkg_config_version_installed('openssl', atleast='0'):
+            library_dirs += [misc.pkg_config_get_var('openssl', 'libdir')]
             libraries += ['ssl', 'crypto']
```

Comment:
It's weird and inconsistent that these entries don't come from pkg-config too [Not to do with your change, but still makes me think!]

Reply:
This actually makes a little bit of sense to me. `ssl` and `crypto` are essentially components of OpenSSL that user chooses to pull in or not. So the line is similar to

```
find_package(Proton 0.34.0 REQUIRED COMPONENTS Core Proactor)
```

where Dispatch also explicitly specifies the names of components, instead of asking the lib for all components available.

Reply:
Well it's just not the usual way you'd use pkg-config! The essential point of it is to tell you what extra flags you need in the compile and link lines to get this library.
It's entirely possible to change what the libraries are called, or add extra dependency libraries, if you look them up using pkg-config. Specifically in this case the extra -lcrypto is not really obvious (except that everyone has just to used to linking it now!)